gwpy/cli/cliproduct.py

```diff
@@ -724,6 +730,50 @@ def run(self):
                 show_error = False
             self.plot_num += 1
 
+    def add_segs(self, args):
+        """ If requested add DQ segments"""
+        std_segments = \
+            [
+                '{ifo}:DMT-GRD_ISC_LOCK_NOMINAL:1',
+                '{ifo}:DMT-DC_READOUT_LOCKED:1',
+                '{ifo}:DMT-CALIBRATED:1',
+                '{ifo}:DMT-ANALYSIS_READY:1'
+            ]
```

Comment:
Can you please reformat this as
```python
        std_segments = [
            '{ifo}:DMT-GRD_ISC_LOCK_NOMINAL:1',
            '{ifo}:DMT-DC_READOUT_LOCKED:1',
            '{ifo}:DMT-CALIBRATED:1',
            '{ifo}:DMT-ANALYSIS_READY:1'
        ]

Reply:
ok

Reply:
Ping @areeda, I don't see this change.

Reply:
a) I don't understand why I don't see email on my PRs including this ping while I seem to get them for everybody else's??

b) This change shows up at https://github.com/areeda/gwpy/blob/segment-add/gwpy/cli/cliproduct.py#L737 I wonder why you didn't see it??

Reply:
@areeda, the link you posted includes identical code to what is currently on the branch, what I want is this:

```diff
diff --git a/gwpy/cli/cliproduct.py b/gwpy/cli/cliproduct.py
index cbbd350e..f237e7fa 100644
--- a/gwpy/cli/cliproduct.py
+++ b/gwpy/cli/cliproduct.py
@@ -735,11 +735,11 @@ class CliProduct(object):
         """ If requested add DQ segments
         """
         std_segments = [
-                '{ifo}:DMT-GRD_ISC_LOCK_NOMINAL:1',
-                '{ifo}:DMT-DC_READOUT_LOCKED:1',
-                '{ifo}:DMT-CALIBRATED:1',
-                '{ifo}:DMT-ANALYSIS_READY:1'
-            ]
+            '{ifo}:DMT-GRD_ISC_LOCK_NOMINAL:1',
+            '{ifo}:DMT-DC_READOUT_LOCKED:1',
+            '{ifo}:DMT-CALIBRATED:1',
+            '{ifo}:DMT-ANALYSIS_READY:1'
+        ]
         segments = list()
         if hasattr(args, 'std_seg'):
             if args.std_seg:
```

Reply:
I see you want indentation changed.  How's this one?

BTW I got an email with your comment did you change something?